data/json/recipes/other/tool.json

```diff
@@ -90,6 +90,28 @@
     ]
   },
   {
+    "result": "pickaxe",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 7,
```

Comment:
Actually ignore for now - I need to review the crafting code and there is no need to hold up your PR for that